v2/support/instances_sample_app/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Instances API Sample App",
+  "author": {
+    "name": "Zendesk",
+    "email": "support@zendesk.com",
+    "url": "https://help.zendesk.com"
+  },
+  "defaultLocale": "en",
+  "private": true,
+  "location": {
+    "support": {
+      "background": "assets/iframe.html",
+      "ticket_sidebar": { "autoLoad": false }
```

Comment:
actually, this could be a sensible default if there is no URL, should we make that change?